src/FileSystemUtilities.js

```diff
@@ -51,10 +51,7 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    ChildProcessUtilities.exec("npm bin", {cwd: __dirname}, (err, npmBinPath) => {
-      if (err) return callback(err);
-      ChildProcessUtilities.spawn(join(npmBinPath.trim(), "rimraf"), [filePath], {}, callback);
-    });
+    ChildProcessUtilities.spawn(require.resolve("rimraf/bin"), [filePath], {}, callback);
```

Comment:
This relies on the internal structure of the `rimraf` package not changing.

Could we combine this with a walk up to `.bin` so we can use the public interface?

Something like:

```javascript
path.join(require.resolve("rimraf"), "../.bin/rimraf")
```